controller/resource.go

```diff
@@ -432,23 +488,51 @@ type TaskState struct {
 	ECSTaskFound bool
 	// ACLTokens are the Consul ACL tokens found for this task id.
 	ACLTokens []*api.ACLTokenListEntry
+	// Consul Service associated with this ECS task
+	Service *api.AgentService
```

Comment:
We should have two service instances per task: one for the service itself, and one for the sidecar proxy.

Reply:
My bad. Missed this one. Thanks for pointing out